### DISCUSSION TITLE
Can you please stop forcing the language of the website or provider a way to change it ?

### DISCUSSION THREAD

**Discussion by 0xbuidler** [rainbow-me/rainbowkit#2229] [Ideas]
Hey guys,

First of all thanks for this amazing products.

That being said, this is low priority but could you please add a way to change the language of the website through the app directly. I'm always changing the url manually to go to the English language.

Thanks and again, this is low priority but still, wanted to let you guys know about this.

**Comment by DanielSinclair** [comment]
@sshmaxime Are you referring to the RainbowKit.com site, or your dApp? We currently detect the browser language preference and default to that for the RainbowKit modal, but dApps can also override this with the provider's `locale` property. https://www.rainbowkit.com/docs/localization
